src/gwt/src/org/rstudio/studio/client/workbench/model/BootInfo.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * BootInfo.java
+ *
+ * Copyright (C) 2009-17 by RStudio, Inc.
+ *
+ * Unless you have received this program directly from RStudio pursuant
+ * to the terms of a commercial license agreement with RStudio, then
+ * this program is licensed to you under the terms of version 3 of the
+ * GNU Affero General Public License. This program is distributed WITHOUT
+ * ANY EXPRESS OR IMPLIED WARRANTY, INCLUDING THOSE OF NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE. Please refer to the
+ * AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
+ *
+ */
+package org.rstudio.studio.client.workbench.model;
+
+import com.google.gwt.core.client.JavaScriptObject;
+
+public class BootInfo extends JavaScriptObject
+{
+   protected BootInfo()
+   {
+   }
+
+   public final native boolean getUseRetinaIcons() /*-{
+      return typeof(this.use_retina_icons) !== "undefined" ? this.use_retina_icons : true;
```

Comment:
Ah, right! Yes, I'm being paranoid making sure that even is the `boot_init` call fails, we can also keep working as expected, will add this.